public/index.html

```diff
@@ -19,20 +18,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
```

Comment:
Pytanie: czemu skasowałaś to title i powyższy manigest?

Reply:
@nettecode Troszkę "siara", ale chyba się machnęłam, jak "sprzątałam" te niektóre początkowe pliki w stylu logo React'a itp. 🙈 Chyba odruch: "o, to się nie przyda, pewnie build zrobi nowy" 😅